docs/plugins/building_a_plugin/_layer_data_guide.md

```diff
@@ -33,13 +38,14 @@ LayerData = Union[Tuple[DataType], Tuple[DataType, LayerProps], FullLayerData]
 where ...
 
 ```python
-from typing import Literal, Protocol, Sequence
+from typing import Dict, Literal, Protocol, Sequence, Tuple, Union
+import numpy as np
 
 LayerTypeName = Literal[
     "image", "labels", "points", "shapes", "surface", "tracks", "vectors"
 ]
 LayerProps = Dict
-DataType = Union[ArrayLike, Sequence[ArrayLike]]
+DataType = Union["ArrayLike", Sequence["ArrayLike"]]
```

Comment:
Actually, let's leave these as is, as I don't think people will be copying this code snippet. This code is just to inform people of what the type aliases are.

Reply:
OK, changed here: https://github.com/napari/docs/pull/367/commits/27e7a908d211cdcab7bb0bd528b6abc2d289a306